src/bot.ts

```diff
@@ -25,16 +25,16 @@ export class Bot {
   @Guard(IsDMChannel)
   @Guard(FromUser(process.env.OWNER_ID))
   async respondToUser(
-    @Option('user', OptionType.USER)
-    userId: string,
-    @Option('message', OptionType.STRING)
+    @SlashOption('user', { type: 'USER', required: true })
+    user: User | GuildMember,
+    @SlashOption('message', { type: 'STRING', required: true })
     message: string,
     interaction: CommandInteraction
   ): Promise<void> {
-    await sendMessageToUser(interaction.client, userId, message);
+    await sendMessageToUser(interaction.client, user.id, message);
```

Comment:
Does this mean that discordx resolves users automatically?

Reply:
yes, if you set option optional then it's

user | guildmember | undefined